x/gamm/pool-models/stableswap/amm_test.go

```diff
@@ -16,22 +16,28 @@ func decApproxEq(t *testing.T, exp sdk.Dec, actual sdk.Dec, errTolerance sdk.Dec
 	require.True(t, diff.LTE(errTolerance), "expected %s, got %s, maximum errTolerance %s", exp, actual, errTolerance)
 }
 
-func TestCFMMInvariant(t *testing.T) {
+func TestCFMMInvariantTwoAssets(t *testing.T) {
 	kErrTolerance := sdk.OneDec()
 
 	tests := []struct {
-		xReserve sdk.Dec
-		yReserve sdk.Dec
-		yIn      sdk.Dec
+		xReserve    sdk.Dec
+		yReserve    sdk.Dec
+		uReserve    sdk.Dec
+		wSumSquares sdk.Dec
```

Comment:
Lets remove these args, and just hard code the 0 and 1

Reply:
Done! Should be in next commit